packages/server-core/src/projects/project/project.hooks.ts

```diff
@@ -259,14 +259,14 @@ const checkIfProjectExists = async (context: HookContext<ProjectService>) => {
   }
 
   const data: ProjectData[] = Array.isArray(context.data) ? context.data : [context.data]
-
   const projectName = data[0].name!
+  const slashIndex = projectName.indexOf('/')
 
-  const orgName = projectName.slice(0, projectName.indexOf('/'))
-
-  const cleanedProjectName = cleanString(projectName.slice(projectName.indexOf('/')))
-
-  context.projectName = `${orgName}/${cleanedProjectName}`.toLowerCase()
+  if (slashIndex > -1) {
+    const orgName = projectName.slice(0, projectName.indexOf('/'))
+    const cleanedProjectName = cleanString(projectName.slice(projectName.indexOf('/')))
+    context.projectName = `${orgName}/${cleanedProjectName}`.toLowerCase()
+  } else context.projectName = projectName.toLowerCase()
```

Comment:
In what scenario this else block will be used?

Reply:
With multitenancy, we're enforcing an <org>/<projectName> naming convention. But in a single-tenant environment, it should be viable to not use org names. When I was testing, I re-used a saved query that was sending the project name without an org name, and it was turning e.g. irpro-stripe into irpro-strip/e.